src/test/java/LoginTests.java

```diff
@@ -18,13 +20,12 @@ public static void LoginValidEmailValidPasswordTest () throws InterruptedExcepti
 
         navigateToPage();
```

Comment:
Methods should be verbs, in mixed case with the **first letter lowercase**, with the first letter of each internal word capitalized.

For example `public static void loginEmptyEmailPasswordTest ()`